packages/ui/src/components/va-file-upload/VaFileUploadUndo/VaFileUploadUndo.vue

```diff
@@ -46,19 +49,15 @@ export default defineComponent({
     const progress = ref(100)
     const {
       undoDuration,
-      deletedFileMessage,
       undoButtonText,
-    } = inject(VaFileUploadKey, {
-      undoDuration: ref(3000),
-      deletedFileMessage: ref('File was successfully deleted'),
-      undoButtonText: ref('Undo'),
-    })
+      deletedFileMessage,
+    } = useStrictInject<VaFileUploadInject>(VaFileUploadKey, INJECTION_ERROR_MESSAGE)
 
     const computedClasses = useBem('va-file-upload-undo', () => ({
       vertical: props.vertical,
     }))
 
-    const undoDurationStyle = computed(() => `${undoDuration.value}ms`)
+    const undoDurationStyle = computed(() => `${undoDuration?.value || 3000}ms`)
```

Comment:
```suggestion
    const undoDurationStyle = computed(() => `${undoDuration.value ?? 0}ms`)
```
If the user doesn't want the duration and sets it to 0, we should use the `v-if` directive with `va-progress-bar` in the template

Reply:
I think here will fit the `undefined` or `null` values for the undo-duration prop, I'll note it in the property description in the docs page.